Radzen.Blazor/wwwroot/Radzen.Blazor.js

```diff
@@ -1449,53 +1449,56 @@ window.Radzen = {
           }
       }, 500);
   },
-  initSideDialogResize: function(handle, sideDialog){
-        const dir = (handle.dataset.dir || 'right').toLowerCase();
-        const cs = window.getComputedStyle(sideDialog);
-        const parent = sideDialog.parentNode;
-        const parentIsFlex = parent && getComputedStyle(parent).display.includes('flex');
-        const toPixels = (v, axis) => {
-            if (!v || v === 'none') return NaN;
-            if (v.endsWith && v.endsWith('px')) return parseFloat(v);
-            if (v.endsWith && v.endsWith('%')) {
-                const base = axis === 'y' ? window.innerHeight : window.innerWidth;
-                const p = parseFloat(v);
-                return Number.isFinite(p) ? (base * p / 100) : NaN;
+  initSideDialogResize: function(handle, sideDialog, options){
+        const normalizeDir = (value) => {
+            if (typeof value === 'string' && value.length) {
+                return value.toLowerCase();
             }
-            const n = parseFloat(v);
-            return Number.isFinite(n) ? n : NaN;
+            if (typeof value === 'number') {
+                const positions = ['right', 'left', 'top', 'bottom'];
+                return positions[value] || 'right';
+            }
+            return 'right';
+        };
+        
+        const dir = normalizeDir(options?.position);
+        
+        const hasMinWidthOption = options && typeof options.resizalbeMinWidth === 'number' && Number.isFinite(options.resizalbeMinWidth);
+        const hasMinHeightOption = options && typeof options.resizableMinHeight === 'number' && Number.isFinite(options.resizableMinHeight);
+        const fallbackMinWidth = hasMinWidthOption ? options.resizalbeMinWidth : 300;
```

Comment:
Better have the default options set in the Blazor code so they are clear in the API reference and code. It would also eliminate the need of these checks and fallbacks e.g.

```
public double ResizableMinWidth { get; set; } = 300;
```